src/toast/ops/sim_ground_utils.py

```diff
@@ -402,6 +402,29 @@ def step_el(
     return new_min_el, new_max_el
 
 
+@function_timer
+def simulate_stare(t_start, t_stop, rate, el, az):
+    """ Return times, azimuth, elevation and intervals for a stare """
+    samples = int((t_stop - t_start) * rate)
+    times = t_start + np.arange(samples) / rate
+    az_sample = np.zeros(samples) + az
+    el_sample = np.zeros(samples) + el
+    # FIXME: do we need a flag bit for stationary telescope?
```

Comment:
When such data is passed to the `AzimuthIntervals` operator, it would be marked as a very long turnaround interval.

Reply:
Probably fine for now, until we actually need to deal with scans like this.